example_school_library_decorator/classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :students
```

Comment:
- [ ] Please use `attr_reader` for `:students`  variable since it belongs to the `Student` class and can not be written directly to in the Animal Class.